app/sdil/models/RegistrationFormData.scala

```diff
@@ -39,11 +39,33 @@ case class RegistrationFormData(rosmData: RosmRegistration,
                                 contactDetails: Option[ContactDetails] = None,
                                 smallProducerConfirmFlag: Option[Boolean] = None) {
 
+  lazy val isSmall: Boolean = {
+    total(packageOwn, copackedVolume) < 1000000
+  }
+
+  lazy val isCopacked: Boolean = {
+    copackedVolume.exists(_.total != 0)
+  }
+
+  lazy val isImporter: Boolean = {
+    imports.contains(true)
+  }
+
+  lazy val isCopacker: Boolean = {
```

Comment:
This is only used once so a val isn't required and I feel having packageCopack.isEmpty is clearer than !packageCopack.nonEmpty